pkg/config/template.go

```diff
@@ -1334,7 +1334,7 @@ const templateStringCrioRuntimeWorkloads = `# The workloads table defines ways t
 # that work based on annotations, rather than the CRI.
 # Note, the behavior of this table is EXPERIMENTAL and may change at any time.
 # Each workload, has a name, activation_annotation, annotation_prefix and set of resources it supports mutating.
-# The currently supported resources are "cpu" (to configure the cpu shares) and "cpuset" to configure the cpuset.
+# The currently supported resources are "cpuperiod" "cpuquota", "cpushares" and "cpuset" to configure the cpuset.
```

Comment:
This sentence now reads as if all of these above are required to configure "cpuset"—is that the case?

Reply:
Good catch, that should not be required, updated wording there to remove that phrasing.